assets/js/index.js

```diff
@@ -31,6 +31,7 @@ $(document).ready(function() {
     });
 
     $('#header__nav__button').on("click", function(){
-        $('#header__nav__links').toggleClass("display");
+      $('#header__nav__links').slideToggle("slow", function () {
```

Comment:
I know but haven't got that far during my learning here, but there is the concept of CSS transisitions. I assume that's what slideToggle is doing under the hood. Have you used CSS transistions @awflwafl or were you planning to?